### PR TITLE
Update FTPWindow.cpp

### PR DESCRIPTION
**Pull request by ricmetal**
Audible notification on uploadComplete! No more will the FTP window take up precious real estate.

### CODE REVIEW

File: src/Windows/FTPWindow.cpp [hunk 1]
```diff
@@ -1037,7 +1037,9 @@ int FTPWindow::OnEvent(QueueOperation * queueOp, int code, void * data, bool isS
 			}
 
 			OutMsg("Upload of %s succeeded.", opuld->GetExternalPath());
-
+			
+			MessageBeep(MB_OK);
```
Comment (ricmetal) [inline src/Windows/FTPWindow.cpp]: would preferably be with an if statement to enable or disable sound

**Comment by ashkulz** [comment]
That's a bad approach ... flashing the window via `::FlashWindow` would be a better approach.

**Comment by ricmetal** [comment]
the only thing bad that i can think of using the sound approach is for people who are constantly saving like every 3 seconds, they'll hear sound. the workaround would be to add a different sound, just a minor subtle beep but there are few sounds to use with MB.

why would you say it is a bad approach?

**Comment by ashkulz** [comment]
Well, I'm one of those ... any audible noise is to be avoided.

**Comment by ricmetal** [comment]
yes, audible noise is not the ideal solution, but know little about c++, and for me, id rather have the audible sound, than have the nppftp window take up any screen estate!

**Pull request closed without merging**